Tests/Recurly/Invoice_Test.php

```diff
@@ -121,4 +121,26 @@ public function testGetInvoicePdf() {
     $result = Recurly_Invoice::getInvoicePdf('1001', 'en-GB', $this->client);
     $this->assertEquals(array('/invoices/1001', 'en-GB'), $result);
   }
+
+  public function testRefundAmount() {
+    $this->client->addResponse('POST', 'https://api.recurly.com/v2/invoices/1001/refund', 'invoices/refund-201.xml');
+    $invoice = Recurly_Invoice::get('1001', $this->client);
+
+    $refund_invoice = $invoice->refundAmount(1000);
+    $this->assertEquals($refund_invoice->subtotal_in_cents, -1000);
+  }
+
+  public function testRefund() {
+    $this->client->addResponse('POST', 'https://api.recurly.com/v2/invoices/1001/refund', 'invoices/refund-201.xml');
+    $invoice = Recurly_Invoice::get('1001', $this->client);
+    $line_items = $invoice->line_items;
+
+    $adjustment_map = function($line_item) {
+      return array("uuid" => $line_item->uuid, "quantity" => $line_item->quantity, "prorate" => false);
+    };
+    $adjustments = array_map($adjustment_map, $line_items);
```

Comment:
Having the caller have to do this transformation seems a little unnecessary. I can see where you'd want to specify values for quantity/prorate but I wonder if we can come up with something a little nicer.